src/components/order-service/order-service.controller.ts

```diff
@@ -257,15 +257,25 @@ export class OrderServiceController implements angular.IController {
   }
 
   private updateBindability() {
-    this.bindStep.hidden = _.size(this.ctrl.applications) < 1;
-    this.reviewStep.allowed = this.bindStep.hidden;
-
-    if (this.bindStep.hidden) {
-      this.ctrl.nextTitle = "Create";
+    var planBindable = _.get(this.ctrl.selectedPlan, "bindable");
```

Comment:
Can you just add a comment here explaining that plan bindable is optional so the explicit `===` checks are required.  Don't want someone looking at this later to try to optimize the checks and then break the behavior :)